scripts/ci/lib.sh

```diff
@@ -198,7 +198,7 @@ push_docs_image() {
     local docs_tag
     docs_tag="$(make --quiet docs-tag)"
 
-    local registries=("docker.io/stackrox" "quay.io/rhacs-eng" "quay.io/stackrox-io")
+    local registries=("quay.io/rhacs-eng" "quay.io/stackrox-io")
 
     for registry in "${registries[@]}"; do
         registry_rw_login "$registry"
```

Comment:
I think an extra line
```sh
oc image mirror "$PIPELINE_DOCS_IMAGE" "${registry}/docs:$(make --quiet tag)"
```
is needed to do the former job of the deleted `Retag docs image with release-version tag in Docker Hub` step in `config.yml`.